src/harness/util.py

```diff
@@ -150,28 +150,31 @@ def getRandomLatLongInPolygon(ppa):
     return getRandomLatLongInPolygon(ppa)
 
 
-def makePpaAndPalRecordsConsistent(ppa_record, pal_records, low_frequency,
-                                   high_frequency, user_id, fcc_channel_id="1"):
-  """Make PPA and PAL object consistent with the inputs
+def makePalRecordsConsistent(pal_records, low_frequency, high_frequency,
+                             user_id, fcc_channel_id="1",
+                             previous_year_date=None, next_year_date=None):
+  """Make Pal object consistent with the inputs
 
     Args:
-      ppa_record: (dictionary) A dictionary containing PPA Record.
       pal_records: (list) A list of PAL Records in the form of dictionary
       which has to be associated with the PPA.
       low_frequency: (number) The Primary Low Frequency for PAL.
       high_frequency: (number) The Primary High Frequency for PAL.
       user_id: (string) The userId from the CBSD.
       fcc_channel_id: (string) The FCC-supplied frequency channel identifier.
-
+      previous_year_date: (string) Pal license start date, generally set as one year
```

Comment:
Why not just call it "start_date" directly? The description can give guidance on how to set it. We might want to test with an expired PAL, and a more general name would make that test case less confusing. (Although I'm not sure such a test is actually in the spec right now...)